PSKoans/Public/Show-Karma.ps1

```diff
@@ -34,6 +34,11 @@ function Show-Karma {
         [switch]
         $Contemplate,
 
+        [Parameter(Mandatory, ParameterSetName = 'OpenFile')]
+        [Alias('Continue')]
+        [switch]
+        $OpenFile,
```

Comment:
I think we're best off refactoring the current implementation into `-Contemplate` itself, and then we can either remove the extra switch or have an `-OpenFolder` instead, which uses the current `-Contemplate` behaviour so users can still do that as well. 🙂

Reply:
All done